lib/validation/validator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# use Validator.new.validator_for_download
+class Validator
+  def validator_for_download(download_flag, extension)
+    case download_flag
+    when 'all'
```

Comment:
I think those values 'all', 'file' etc should be symbols - `:all`, :file` etc...

Read about symbols in ruby - it's rather uniq and intersting difference between string and symbol and as far as I'm aware - there is no such think in python as symbols